package.json

```diff
@@ -2,11 +2,10 @@
   "name": "@uploadcare/api",
   "version": "1.0.0-alpha",
   "description": "JavaScript library for work with Uploadcare API",
-  "main": "src/uploadcare.cjs.js",
-  "module": "dist/uploadcare.esm.js",
-  "browser": "dist/uploadcare.umd.js",
+  "main": "dist/uploadcare.cjs.js",
+  "module": "index.js",
```

Comment:
are you sure? I think esm bundle for module and iife for browser is ok

Reply:
@Zmoki yep, `browser` filed is not for IIFE, it is used by bundlers to use as entry point for browsers. It is needed if we have some browser-specific difference between bundles. But we have not them so we do not need `browser` property.

Reference: https://github.com/defunctzombie/package-browser-field-spec

Reply:
but `module` is the esm bundle, not the source code

Reply:
@Zmoki `index.js` is not source code. It is entry point that exporting modules from bundles.

```
import * as UploadAPI from './dist/upload.esm'

export {UploadAPI}
```

Also we have `upload.js` which exporting only `UploadAPI`:

```
export * from './dist/upload.esm'
```